cli/cmd/metrics.go

```diff
@@ -85,11 +87,20 @@ func newCmdMetrics() *cobra.Command {
 			var buf bytes.Buffer
 			for i, result := range results {
 				content := fmt.Sprintf("#\n# POD %s (%d of %d)\n#\n", result.pod, i+1, len(results))
-				if result.err == nil {
+				switch {
+				case result.err != nil:
+					content += fmt.Sprintf("# ERROR: %s\n", result.err)
+				case result.err == nil && options.obfuscate:
+					obfuscatedMetrics, err := obfuscateMetrics(result.metrics)
+					if err != nil {
+						content += fmt.Sprintf("# ERROR %s\n", err)
+					} else {
+						content += string(obfuscatedMetrics)
+					}
+				case result.err == nil && !options.obfuscate:
```

Comment:
We should be able to simplify this `switch` a little

```suggestion
				switch {
				case result.err != nil:
					content += fmt.Sprintf("# ERROR: %s\n", result.err)
				case options.obfuscate:
					obfuscatedMetrics, err := obfuscateMetrics(result.metrics)
					if err != nil {
						content += fmt.Sprintf("# ERROR %s\n", err)
					} else {
						content += string(obfuscatedMetrics)
					}
				default:
					content += string(result.metrics)
				}
```

Reply:
Ah yes thanks for catching this, I implemented the suggestion in fb44bef